moveit_core/urdfdom_compatibility.h.in

```diff
@@ -0,0 +1,78 @@
+/*********************************************************************
```

Comment:
This file is not a usual include, but it's a template to generate the actual include. Hence, it should reside together with the CMakeLists.txt.